### PR TITLE
LPD-39837 Enhancements on the display menu, on Apps Page, Solutions Page, app Details Page

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
ci:test:sf

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layouts/01_app_marketplace/page-definition.json [hunk 1]
```diff
@@ -697,7 +594,7 @@
 																	"maxTerms": "10",
 																	"order": "count:desc",
 																	"parameterName": "Liferay Platform Offering",
-																	"vocabularyIds": "[$TAXONOMY_VOCABULARY_ID:Marketplace Liferay Platform Offering$]"
+																	"vocabularyIds": "33756"
```
Comment (eduardo-diniz) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layouts/01_app_marketplace/page-definition.json:597]: add var
```suggestion
																	"vocabularyIds":[$$]

```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/ddm-templates/marketplace-apps-categories/ddm-template.ftl [hunk 2]
```diff
@@ -3,16 +3,29 @@
 		font-size: MEDIUM;
 	}
 
+	.app-category {
+		display:block;
+		flex: 1;
+		max-width:200px;
+		overflow: hidden;
+		text-overflow: ellipsis;
+		white-space: nowrap;
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/ddm-templates/marketplace-apps-categories/ddm-template.ftl:12]:
```suggestion
		display: block;
		flex: 1;
		max-width: 200px;
		overflow: hidden;
		text-overflow: ellipsis;
		white-space: nowrap;
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/ddm-templates/marketplace-apps-categories/ddm-template.ftl [hunk 3]
```diff
@@ -91,7 +104,7 @@
 				</#if>
 
 				<#if type?has_content && icon?has_content>
-					<div class="app-product-type border border-radius-small d-flex mb-1 mr-2 px-1 rounded">
+					<div class="app-product-type border border-radius-small d-flex mb-1 mr-2 px-2 rounded-lg align-items-center">
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/ddm-templates/marketplace-apps-categories/ddm-template.ftl:107]: Sort classes alphabetical 

```suggestion
					<div class="app-product-type border border-radius-small d-flex mb-1 mr-2 px-2 rounded-lg align-items-center">
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/display-page-templates/app-detail/page-definition.json [hunk 4]
```diff
@@ -178,7 +175,7 @@
 																"definition": {
 																	"fragment": {
 																		"key": "text",
-																		"siteKey": "[$GROUP_KEY$]"
+																		"siteKey": "Marketplace"
```
Comment (eduardo-diniz) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/display-page-templates/app-detail/page-definition.json:178]: add Var
```suggestion
																		"siteKey": "[$GROUP_KEY$]"
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/display-page-templates/app-detail/page-definition.json [hunk 5]
```diff
@@ -75,7 +76,7 @@
 													"customCSS": ".[$FRAGMENT_CLASS$] img {\n    height: 90px;\n    object-fit: contain;\n    width: 90px;\n}",
 													"fragment": {
 														"key": "image",
-														"siteKey": "[$GROUP_KEY$]"
+														"siteKey": "Marketplace"
```
Comment (eduardo-diniz) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/display-page-templates/app-detail/page-definition.json:79]: add Var
```suggestion
														"siteKey": "[$GROUP_KEY$]"
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master-private/css.css [hunk 6]
```diff
@@ -1,13 +1,55 @@
 /*Source Sans 3*/
 @font-face {
 	font-display: swap;
-	font-family: 'Source Sans 3';
+	font-family: "Source Sans 3";
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master-private/css.css:4]: As before

```suggestion
	font-family: 'Source Sans 3';
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master-private/css.css [hunk 7]
```diff
@@ -1,13 +1,55 @@
...
+.child-menu.dropdown-menu {
+	border-radius: 8px !important;
+	padding: 0px !important;
+}
+
+.card.main-image {
+	border-radius:10px !important;
+}
+
+.product-gallery .arrow.next{
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master-private/css.css:47]:
```suggestion
.product-gallery .arrow.next {
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master-private/css.css [hunk 8]
```diff
@@ -1,13 +1,55 @@
...
+	border-bottom-right-radius: 10px;
+	border-top-right-radius: 10px;
+}
+
+.product-gallery .arrow.prev{
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master-private/css.css:52]:
```suggestion
.product-gallery .arrow.prev {
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master-private/css.css [hunk 9]
```diff
@@ -1,13 +1,55 @@
...
+	font-weight: none !important;
+}
+
+.lfr-nav-item.nav-item.selected.active .nav-link {
+	border-bottom: #004AD7 solid 3px !important;
+	color: #004ad7 !important;
+}
+
+.lfr-nav-item.nav-item.dropdown:has(.child-menu.dropdown-menu .active.selected)
+	.nav-link {
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master-private/css.css:32]:
```suggestion
.lfr-nav-item.nav-item.dropdown:has(.child-menu.dropdown-menu .active.selected) .nav-link {
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master/css.css [hunk 10]
```diff
@@ -1,13 +1,55 @@
 /*Source Sans 3*/
 @font-face {
 	font-display: swap;
-	font-family: 'Source Sans 3';
+	font-family: "Source Sans 3";
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master/css.css:4]: As before

```suggestion
	font-family: 'Source Sans 3';
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master/css.css [hunk 11]
```diff
@@ -1,13 +1,55 @@
...
+.lfr-nav-item.nav-item.selected.active .nav-link {
+	border-bottom: #004AD7 solid 3px !important;
+	color: #004ad7 !important;
+}
+
+.lfr-nav-item.nav-item.dropdown:has(.child-menu.dropdown-menu .active.selected)
+	.nav-link {
+	border-bottom: #004AD7 solid 3px !important;
+	color: #004ad7 !important;
+
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master/css.css:35]:
```suggestion
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master/css.css [hunk 12]
```diff
@@ -1,13 +1,55 @@
...
+.child-menu.dropdown-menu {
+	border-radius: 8px !important;
+	padding: 0px !important;
+}
+
+.card.main-image {
+	border-radius:10px !important;
+}
+
+.product-gallery .arrow.next{
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master/css.css:47]:
```suggestion
.product-gallery .arrow.next {
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master/css.css [hunk 13]
```diff
@@ -1,13 +1,55 @@
...
+	border-bottom-right-radius: 10px;
+	border-top-right-radius: 10px;
+}
+
+.product-gallery .arrow.prev{
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master/css.css:52]:
```suggestion
.product-gallery .arrow.prev {
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/style-books/marketplace/frontend-tokens-values.json [hunk 14]
```diff
@@ -49,7 +49,7 @@
 	},
 	"gray100Color": {
 		"cssVariableMapping": "gray-100",
-		"value": "#F4F6F9"
+		"value": " #F7F7F8"
```
Comment (eduardo-diniz) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/style-books/marketplace/frontend-tokens-values.json:52]:
```suggestion
		"value": "#F7F7F8"
```

Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/style-books/marketplace/frontend-tokens-values.json:52]:
```suggestion
		"value": "#F7F7F8"
```

**Review by eduardo-diniz — CHANGES_REQUESTED** [review]

**Review by kevenleone — CHANGES_REQUESTED** [review]
Thank you!!!

I left some suggestions

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:test:sf

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master-private/css.css [hunk 15]
```diff
@@ -4,10 +4,51 @@
...
+
+.lfr-nav-item.nav-item.selected.active .nav-link {
+	border-bottom: #004AD7 solid 3px !important;
+	color: #004ad7 !important;
+}
+
+.lfr-nav-item.nav-item.dropdown:has(.child-menu.dropdown-menu .active.selected) .nav-link {
+	border-bottom: #004AD7 solid 3px !important;
+	color: #004ad7 !important;
+
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master-private/css.css:34]:
```suggestion
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master/css.css [hunk 16]
```diff
@@ -4,10 +4,51 @@
...
+
+.lfr-nav-item.nav-item.selected.active .nav-link {
+	border-bottom: #004AD7 solid 3px !important;
+	color: #004ad7 !important;
+}
+
+.lfr-nav-item.nav-item.dropdown:has(.child-menu.dropdown-menu .active.selected) .nav-link {
+	border-bottom: #004AD7 solid 3px !important;
+	color: #004ad7 !important;
+
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master/css.css:34]:
```suggestion
```

**Review by kevenleone — APPROVED** [review]

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:stop

**Comment by kevenleone** [comment]
ci:forward:force

**Pull request closed without merging**